dsgrid/config/association_tables.py

```diff
@@ -31,6 +31,11 @@ class AssociationTableRecordModel(DSGBaseModel):
         title="to_id",
         description="Destination mapping",
     )
```

Comment:
General question: Is this the data model for dimension mappings, dimension associations, or both?

Reply:
Looks like it's just for dimension_mappings. I know it's confusing. Association_tables (DimensionAssociations) are just in dict form:
https://github.com/dsgrid/dsgrid/blob/1d4ecde7aadf73da1fae6e49b76c09e22c09e68c/dsgrid/submit/submit_dataset.py#L67-L70

And if I am seeing this correctly, DimensionAssociations are not checked for completeness:
https://github.com/dsgrid/dsgrid/blob/98e95326f83a66d7327962374defbaa2451acf60/dsgrid/config/dimension_associations.py

Reply:
Should we change the data model class name to MappingTableRecordModel (or DimensionMappingTableRecordModel)? @lixiliu @mooneyme